### PR TITLE
Feat/design/form

### PR DESCRIPTION
**Pull request by zanskorja**
Merging changes frrom styling to development

### CODE REVIEW

File: resources/views/admin/subject/list.blade.php [hunk 1]
```diff
@@ -1,26 +1,17 @@
 @extends('layout')
 
 @section('content')
+<div class="box">
     <h1>Obstoječi predmeti</h1>
 
-    @if (session('message'))
```
Comment (majzabukovnik) [inline resources/views/admin/subject/list.blade.php:6]: Why is this deleted

**Review by majzabukovnik — CHANGES_REQUESTED** [review]

File: resources/views/admin/subject/list.blade.php (hunk 1 above)
Comment (zanskorja) [reply resources/views/admin/subject/list.blade.php:6]: It was moved further down

**Review by majzabukovnik — APPROVED** [review]
